ports/stm32f4/common-hal/pulseio/PWMOut.c

```diff
@@ -113,12 +114,11 @@ pwmout_result_t common_hal_pulseio_pwmout_construct(pulseio_pwmout_obj_t* self,
     bool first_time_setup = true;
 
     for (uint i = 0; i < tim_num; i++) {
-        mcu_tim_pin_obj_t l_tim = mcu_tim_pin_list[i];
-        uint8_t l_tim_index = l_tim.tim_index - 1;
-        uint8_t l_tim_channel = l_tim.channel_index - 1;
```

Comment:
I'd prefer if it was more like this. I think it will work if `l_tim` is a pointer rather than a copy. So it'd be `mcu_tim_pin_obj_t *l_tim = &mcu_tim_pin_list[i];` and then `l_tim->` instead of `l_tim.` below.